js/cards.js

```diff
@@ -254,6 +307,12 @@ export function renderCards(cards, htmlLocation, replace = false) {
   }
   // setup hover for all cards
   setupHover();
+
+  // only on the gacha page, but we want to do this after cards are added
+  if(document.getElementById("gacha-controls") !== null){
+    // update the motion settings based on the checkbox
+    updateGachaMotion();
+  }
```

Comment:
Is this line strictly needed if the checkbox setting isn't saved in localStorage and always starts unchecked?

Reply:
If it's not there it doesn't apply the "remove-motion" class to the cards upon the cards being generated.